usr/share/rear/init/default/001_verify_config_arrays.sh

```diff
@@ -0,0 +1,22 @@
+# ensure that all array variables defined in default.conf are still array variables after
+# reading the user configuration, fixes common user config error: https://github.com/rear/rear/issues/2930
+
+array_variables=($(
```

Comment:
use 'local' for local variables,
see "Variables" in
https://github.com/rear/rear/wiki/Coding-Style

Reply:
I thought about that but `local` is supposed to work only in functions and this is not in a function...??

Reply:
See the section "Variables" in
https://github.com/rear/rear/wiki/Coding-Style
```
All scripts in usr/share/rear/ are
sourced from within the Source() function
so you can and should use 'local'
for local variables in scripts.
```
At least this is my understanding how it works
(but too often my understanding is incomplete
 and what actualy happens is a bit different).

Reply:
Ahhh, good point and excellent thinking! Thank you!